software/fluidd.yml

```diff
@@ -0,0 +1,13 @@
+name: Fluidd
+website_url: https://docs.fluidd.xyz/
+description: Fluidd is a lightweight & responsive user interface for Klipper, the 3D printer firmware.
+licenses:
+  - GPL-3.0
+platforms:
+  - TypeScript
+tags:
+  - 3D printing
+source_code_url: https://github.com/fluidd-core/fluidd
```

Comment:
> For Klipper

```suggestion
tags:
  - 3D printing
source_code_url: https://github.com/fluidd-core/fluidd
depends_3rdparty: yes
```

Reply:
Should be Klipper be added in this file?

Reply:
I don't think `depends_3rdparty: yes` applies since Klipper is also Free Software: https://www.klipper3d.org/